docs/querying/sql-functions.md

```diff
@@ -1414,11 +1562,32 @@ Returns the number of the row within the window starting from 1.
 
 ## RPAD
 
-`RPAD(<CHARACTER>, <INTEGER>, [<CHARACTER>])`
+Returns a string of size `length` from `expr`. When the length of `expr` is less than `length`, right pads `expr` with `chars`, which defaults to the space character. Truncates `expr` to `length` if `length` is shorter than the length of `expr`.
 
-**Function type:** [Scalar, string](sql-scalar.md#string-functions)
+* **Syntax:** `RPAD(expr, length[, chars])`
+* **Function type:** Scalar, string
+
+<details><summary>Example</summary>
+
+The following example inserts `+` characters as padding to the right of the `OriginState` column from the `flight-carriers` datasource, making the return expression a length of `5` characters.
+
+```sql
+SELECT 
+  "OriginState" AS "origin_state",
+  RPAD("OriginState", 5, '+') AS "add_right_padding"
```

Comment:
can we also add a TRUNCATE example?

Reply:
Extended the same example so that it would cover adding padding, truncating, and no change